tests/main/svcs-disable-install-hook/task.yaml

```diff
@@ -0,0 +1,12 @@
+summary: Check that `snapctl stop --disable` actually stops services on install
+
+execute: |
+    # shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB"/snaps.sh
+    install_local test-snapd-svcs-disable-install-hook
+
+    for service in simple forking; do
+        echo "Verify that the $service service isn't running"
+        snap services | MATCH "test-snapd-svcs-disable-install-hook\\.$service\\s+disabled\\s+inactive"
+    done
+    
```

Comment:
Stray tab?

Reply:
It appears that way, do you want me to delete it?

Reply:
Please do.